packages/ui/src/components/va-select/VaSelect.vue

```diff
@@ -652,8 +687,8 @@ export default defineComponent({
       focusPreviousOption,
       focusNextOption,
       showDropdownContentComputed,
-      showDropdown,
-      hideDropdown,
+      handleDropdownOpen,
+      handleDropdownClose,
```

Comment:
:D

Actually, i tried to handle `open/close-dropdown` events from `VaSelectContent` and names of these two methods were conflicting. After i rejected this idea, i've decided to keep new names, because they're more informative for me - not just open, but proceeding additional related logic.